packages/@o3r-training/training-tools/tsconfig.build.json

```diff
@@ -4,10 +4,11 @@
     "sourceMap": true,
     "incremental": true,
     "composite": true,
-    "rootDir": "src",
+    "rootDir": ".",
```

Comment:
I had to change this for the cli to work (the import of `@o3r/training-tools` has been changed to relative path in `extract-folder-structure` on `main`)